packages/brokeneck-react/src/hooks/useFields.js

```diff
@@ -94,17 +109,27 @@ export default function useFields(typeName) {
   return fields
 }
 
-function formatField(field, value, fieldMetadata) {
+function formatField(
+  field,
+  value,
+  fieldMetadata,
+  theme = { palette: {} },
```

Comment:
this default value is for some reason? in case we don't have a theme or something?